### DISCUSSION TITLE
Keymap with given parameter

### DISCUSSION THREAD

**Discussion by jabcross** [helix-editor/helix#11541] [Q&A]
I'd like to know if it's possible to create a keymap for a command that takes a parameter, such as `"` or `f`, where the parameter is already baked into the command. For instance, if I want a keybind that will be equivalent to typing `"k` to select register "K", for instance.

**Comment by kirawi** [comment]
This is possible on master https://github.com/helix-editor/helix/pull/4709

**Reply by jabcross** [reply]
So it is. Is there a way to do the same for commands that don't have keybindings? If I didn't have `f` mapped to `find_next_char` but wanted to write a macro that uses it anyways, for instance.

**Reply by kirawi** [reply]
I don't think so since it sounded like there were issues with that, but \cc @the-mikedavis

**Reply by the-mikedavis** [reply]
Yeah you can't currently combine macros with regular or typed commands in a sequence. You need to have the keys bound like in a macro
